src/main/java/com/mentoring/configuration/S3Configuration.java

```diff
@@ -0,0 +1,26 @@
+package com.mentoring.configuration;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class S3Configuration {
+
+    @Value("${s3.bucket-name}")
+    private String bucketName;
+    private static final String AWS_ACCESS_KEY_ID = "access-key";
+    private static final String AWS_SECRET_ACCESS_KEY = "access-key";
```

Comment:
If you need credentials, just let me know.

Reply:
It's possible to use the ConfigurationProperties annotation, able to get any properties from the properties file.


```
/**
 * Properties having parameters required on retrieving the transactions from Calculation Service.
 */
@Getter
@AllArgsConstructor
@ConstructorBinding
@ConfigurationProperties(prefix = "ppro.calculation")
public class CalculationProperties {
    private final String host;
    private final String user;
    private final String password;
    private final String authorization;
    private final Integer timeout;
    private final String processedEndpoint;
    private final String country;
}
```